.github/workflows/delete-pr-deployment.yaml

```diff
@@ -66,3 +69,29 @@ jobs:
             -a ${{ inputs.argo-base-app }} \
             --suffix ${{ steps.prepare.outputs.argocd-suffix }} \
             -b ${{ inputs.dev-gitops-base-branch }}
+
+  status-badge:
+    runs-on: ubuntu-latest
+    if: "contains(github.event.pull_request.body, 'App Deployment') && (!contains(github.event.pull_request.labels.*.name, 'deploy') || github.event.action == 'closed')"
+    timeout-minutes: 5
+    steps:
+      - name: Remove status
+        uses: actions/github-script@v5
+        with:
+          script: |
+            const pr = await github.rest.pulls.get({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: context.issue.number
+            });
+
+            let i = 0;
+            const prBody = pr.data.body;
+            const newBody = prBody.split('\n').filter(line => i++ > 5).join('\n');
+            github.rest.pulls.update({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: context.payload.number,
+              body: newBody
+            });
+          github-token: ${{ secrets.github-token }}
```

Comment:
I would move this into a composite which can be re-used here and for PR deployments in gitops repos.

Alternatively we could also leave the badge in the description of the PR, knowing that it will show something odd for closed PRs.

In any case should we remove the badge once the label `deploy` is removed from the PR?